cpp/oneapi/dal/algo/louvain/detail/vertex_partitioning_ops.hpp

```diff
@@ -48,7 +48,44 @@ struct vertex_partitioning_ops {
     using result_t = vertex_partitioning_result<task_t>;
     using descriptor_base_t = descriptor_base<task_t>;
 
-    void check_preconditions(const Descriptor &desc, input_t &input) const {}
+    void check_preconditions(const Descriptor &desc, input_t &input) const {
+        using msg = dal::detail::error_messages;
+        if (desc.get_accuracy_threshold() < 0) {
+            throw invalid_argument(msg::accuracy_threshold_lt_zero());
```

Comment:
minor: in the message all terms but lz are in the full form; looks like inconsistent style

Reply:
The use of abbreviations lt and gt is aligned with error messages for other algos, since there is a requirement to use LaTeX-like shortening for mathematical operations